modules/oc-mirror-mirror-to-disk.adoc

```diff
@@ -29,6 +29,7 @@ Do not delete or modify the metadata that is generated by the oc-mirror plugin.
 * You have installed the OpenShift CLI (`oc`).
 * You have installed the `oc-mirror` CLI plugin.
 * You have created the image set configuration file.
+* You have set the umask parameter to `0022` on the operating system that uses oc-mirror.
```

Comment:
have to set the umask

Reply:
This has been verified with proof reader through other PR.